frontend/bank-transfer-ui/src/styles.css

```diff
@@ -10,9 +10,49 @@
   font-style: normal;
 }
 
-body {
-  min-height: 100vh !important;
+html, body {
+  height: 100% !important;
+}
+.wrapper {
+  flex: 1;
+  display: flex;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
 }
+
 .content {
-  padding: 20px 0;
+  flex: 1;
+  padding: 50px 0 100px 0;
+  background: rgb(255, 255, 255);
+}
+#main {
+  min-height: 100vh !important;
+  display: flex;
+  flex-direction: column;
+}
+#main > :nth-child(3) {
```

Comment:
Stworzyłeś klasę .wrapper, która robi dokładnie to co jest tutaj. Można się tego pozbyć